webClient/src/app/shared/editor-control/editor-control.service.ts

```diff
@@ -506,6 +533,56 @@ export class EditorControlService implements ZLUX.IEditor, ZLUX.IEditorMultiBuff
       this.openDirectory.next(results.directory);
     });
   }
+
+
+  doSavingDataset(context: ProjectContext, requestUrl: string, _activeFile: ProjectContext, results: any, isUntagged: boolean,
+    _observer: Observer<void>, _observable: Observable<void>) {
+
+      this.snackBar.open(`Attempting to save Dataset ${results}`, 'Close', { duration: MessageDuration.Short, panelClass: 'center' });
```

Comment:
Check that the formatting is correct with other snackbar content names (for example: Do we write files as 'file_name' or "file_name" or file_name etc) I think in USS UX, we also don't say "File file_name" because it is assumed that a user will understand File vs Dataset depending on what tab they're on

Small things Ik, but just check that the Editor looks complete and not like our USS & Dataset sides look like two components developed by different teams